core/domain/config_domain.py

```diff
@@ -208,7 +212,13 @@ class ConfigProperty:
     - whitelisted_email_senders.
     """
 
-    def __init__(self, name, schema, description, default_value):
+    def __init__(
+        self,
+        name: str,
+        schema: Dict[str, Any],
+        description: str,
+        default_value: Any
+    ) -> None:
```

Comment:
Please add a comment explaining why `schema` and `default_value` are `None`.

Reply:
for __init__ : it's a constructor, all it does to populate the values and return nothing ( at least here).
and for schema: it accepts schemas which are of types dictionary with strings as key and with various kinds of values like  https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L115
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L54
for default_value: it can be of any of the type ( bool, list[str], float, int, etc)

Reply:
Ah, sorry I meant explain why `Any` is used. Please add the comments explaining it in the code.

Reply:
done!

Reply:
Done!